### PR TITLE
Starts Swagger/OpenAPI 3

### PR DESCRIPTION
**Pull request by jnallard**
I created a new branch called swagger-3 which can be used to capture our progress towards Swagger/OpenAPI 3. 

This change is very minimal - I created this PR just to bring attention to the new branch. 

Is AppVeyor going to push NuGet packages when the swagger-3 branch is merged into? We don't want to push any packages yet.

### CODE REVIEW

**Comment by yahehe** [comment]
Not sure if Appveyor will publish or not, I would hope not. I can check later.
If we disable package creation on the branch then that would effectively disable publishing

**Comment by yahehe** [comment]
Honestly we might even want to just create a new Nancy.OpenAPI repository and start there (we can clone this over so we aren't starting from scratch)

**Comment by jnallard** [comment]
I don't see the point in starting a new repository.

**Comment by yahehe** [comment]
My main thought process is that continuing with the name Nancy.Swagger will be confusing to consumers of the package looking to use OpenAPI 3. I'm probably over-thinking that point.
That said I guess it wouldn't make sense to lock down the Nancy.OpenAPI namespace now since we don't have it implemented yet (someone else might get to it first), so +1 for developing on the feature branch

**Comment by yahehe** [comment]
Only master and 1.4.3-stable are configured for builds, only master is configured for publishing. I can configure this branch to build but not publish

File: src/Swagger.ObjectModel/SwaggerRoot.cs [hunk 1]
```diff
@@ -29,8 +29,8 @@ public class SwaggerRoot : SwaggerModel
         /// <remarks>
         /// The value MUST be an existing Swagger specification version.
         /// </remarks>
-        [SwaggerProperty("swagger", true)]
-        public string SwaggerVersion
+        [SwaggerProperty("openapi", true)]
+        public string OpenAPI
```
Comment (yahehe) [inline src/Swagger.ObjectModel/SwaggerRoot.cs]: OpenAPIVersion?

Comment (jnallard) [reply src/Swagger.ObjectModel/SwaggerRoot.cs]: I guess we can name it to be more clear - I was just going with the same name the specification used, but just changed the capitalization.

**Comment by yahehe** [comment]
Don't think I turned off publishing correctly...

**Comment by jnallard** [comment]
Did you delete the nuget packages?

**Comment by yahehe** [comment]
No, I will

**Pull request merged**